sormas-backend/src/main/java/de/symeda/sormas/backend/common/DeletableAdo.java

```diff
@@ -29,4 +36,21 @@ public void setDeleted(boolean deleted) {
 		this.deleted = deleted;
 	}
 
+	@Enumerated(EnumType.STRING)
+	public DeletionReason getDeleteReason() {
```

Comment:
Getters and setters need to be changed here as well

Reply:
changed